lib/sycamore/sycamore/functions/document.py

```diff
@@ -81,16 +82,17 @@ class DrawBoxes:
                 .map_batch(DrawBoxes, f_constructor_args=[font_path])
     """
 
-    def __init__(self, font_path: str, default_color: str = "blue"):
+    def __init__(self, font_path: str, default_color: str = "blue", draw_table_cells: bool = True):
         self.font = ImageFont.truetype(font_path, 20)
         self.color_map = {
             "Title": "red",
             "NarrativeText": "blue",
             "UncategorizedText": "blue",
             "ListItem": "green",
-            "Table": "orange",
+            "table": "orange",
```

Comment:
this intentional or typo?

Reply:
It turned out the TableElement had the type set to lowercase, so I changed it to lower case here. In hindsight I think I should make it upper case everywhere to match other types. I'll make that change.

Reply:
Hmm. Actually maybe I was wrong that we standardize on upper-case. There are still a bunch of tests that appear to assume lowercase. Maybe I'll leave this for now, and we can see about cleaning this up holistically.